File: Udemy/Algoritmos e Logica de Programacao/Python/Estrutura Sequencial/ProblemaDuracao.py
'''Problema "duracao"
Fazer um programa para ler uma duração de tempo em segundos, daí imprimir na tela esta duração no
formato horas:minutos:segundos.'''

duracao: int; horas: int; minutos: int; segundos: int; resto: int;

duracao = int(input("Digite a duracao em segundos: "))

horas = duracao // 3600
resto = duracao % 3600
minutos = resto // 60
segundos = resto % 60

print(f"{horas}:{minutos}:{segundos}")

File: Udemy/Algoritmos e Logica de Programacao/C++/Estrutura Sequencial/ProblemaDuracao.cpp
/*
Problema "duracao"
Fazer um programa para ler uma duração de tempo em segundos, daí imprimir na tela esta duração no
formato horas:minutos:segundos.
*/


#include <iostream>
#include <iomanip>

using namespace std;

int main(){

	int duracao, horas, minutos, segundos, resto;

	cout << "Digite a duracao em segundos: ";
	cin >> duracao;	
	
	horas = duracao / 3600;
	resto = duracao % 3600;
	minutos = resto / 60;
	segundos = resto % 60;
	
	cout << horas << ":" << minutos << ":" << segundos;
	
    return 0;
}
